osu.Game/Screens/Play/PlayerTouchInputHandler.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Game.Configuration;
+using osu.Game.Overlays;
+using osu.Game.Overlays.OSD;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Screens.Play
+{
+    public partial class PlayerTouchInputHandler : Component
```

Comment:
I'm not 100% on this class name, as it sounds like it might do more than it's actually doing. How would `PlayerTouchInputDetector` sit with you?

Reply:
Applied in https://github.com/ppy/osu/pull/25348/commits/97fee6143c7b81d169452396a490b8293b521527